server/controllers/notebook.js

```diff
@@ -196,3 +196,24 @@ exports.MoveNotebook = function (req, res) {
   // TODO: refactor to use findOneAndUpdate
   Notebook.findById(notebookId, handleFindNotebook)
 }
+
+//PUT request for Starred Notepage
+exports.UpdateStarredNotebooks = function(req,res){
+	const userId = req.params.userId
+	
+	if(!userId){
+		res.status(401).send("Failed to provide a user id")
+	} 
+	const handleUpdateStarredNotebooks = function(error, notepages){
+		if(error){
+			console.log(error)
+			res.status(500).send("Error getting favorite notepages")
+		}
+		else{
+			const noteDetails = notebooks.details()
+			res.status(200).send(noteDetails)
+		}
+	}
+	const query = Notebook.findAndUpdate(({username: req.body.username},{$set: {starred: req.body.starred}}, handleUpdateStarredNotebooks)
```

Comment:
You don't need exec and also it's `Notebook.findOneAndUpdate` which already executes the code.